app/models/user.rb

```diff
@@ -43,22 +43,22 @@ class User < ApplicationRecord
   acts_as_followable
   acts_as_follower
 
-  has_one :notification_setting, class_name: "Users::NotificationSetting", dependent: :destroy
-  has_one :setting, class_name: "Users::Setting", dependent: :destroy
+  has_one :notification_setting, class_name: "Users::NotificationSetting", dependent: :delete
+  has_one :setting, class_name: "Users::Setting", dependent: :delete
 
   has_many :access_grants, class_name: "Doorkeeper::AccessGrant", foreign_key: :resource_owner_id,
                            inverse_of: :resource_owner, dependent: :delete_all
   has_many :access_tokens, class_name: "Doorkeeper::AccessToken", foreign_key: :resource_owner_id,
                            inverse_of: :resource_owner, dependent: :delete_all
   has_many :affected_feedback_messages, class_name: "FeedbackMessage",
                                         inverse_of: :affected, foreign_key: :affected_id, dependent: :nullify
-  has_many :ahoy_events, class_name: "Ahoy::Event", dependent: :destroy
-  has_many :ahoy_visits, class_name: "Ahoy::Visit", dependent: :destroy
-  has_many :api_secrets, dependent: :destroy
+  has_many :ahoy_events, class_name: "Ahoy::Event", dependent: :delete_all
```

Comment:
The `dependent` clause here is actually unnecessary. The `ahoy_events` table has this FK constraint:

```
"fk_rails_f1ed9fc4a0" FOREIGN KEY (user_id) REFERENCES users(id) ON DELETE CASCADE
```